sormas-ui/src/main/java/de/symeda/sormas/ui/utils/SidePanelLayout.java

```diff
@@ -0,0 +1,56 @@
+package de.symeda.sormas.ui.utils;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.vaadin.ui.Component;
+import com.vaadin.ui.CustomLayout;
+
+public class SidePanelLayout extends CustomLayout {
```

Comment:
```suggestion
public class LayoutWithSidePanel extends CustomLayout {
```

=> "SidePanelLayout" sounds like it IS a side panel, not like it includes one

Reply:
changed